### PR TITLE
Add autoscaler to build09

### PR DESCRIPTION
**Pull request by bear-redhat**

### CODE REVIEW

File: clusters/build-clusters/build09/openshift-machine-api/autoscaler-worker.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+metadata:
+  name: build09-x99t2-worker-us-east-2a
+  namespace: openshift-machine-api
+spec:
+  maxReplicas: 10
+  minReplicas: 1
+  scaleTargetRef:
+    apiVersion: machine.openshift.io/v1beta1
+    kind: MachineSet
+    name: build09-x99t2-worker-us-east-2a
```
Comment (danilo-gemoli) [inline clusters/build-clusters/build09/openshift-machine-api/autoscaler-worker.yaml:12]: Where do the names `build09-x99t2-worker-us-east-*` come from? Are they auto-generated?

Comment (bear-redhat) [reply clusters/build-clusters/build09/openshift-machine-api/autoscaler-worker.yaml:12]: yes thats the default naming

**Pull request merged**